package.json

```diff
@@ -65,7 +65,8 @@
     "build:webpack-release": "webpack --config webpack.release.config.js",
     "build:webpack-e2e-app": "webpack target/e2e/app/app-using-osw-entry.js target/js/app-bundle.js",
     "prestart": "yarn build:dev",
-    "prepublishOnly": "yarn build:release"
+    "prepublishOnly": "yarn build:release",
+    "authenticator": "dyson playground/mocks/authenticator/ 6512"
```

Comment:
nit:
`authenticator` is not specific enough. Maybe something like `mock-authenticator`, `start-authenticator-mocks`.

Reply:
The path contains `mocks`, so I think it implies that it is the mock for authenticator. I can change it to `playground/mocks/start-authenticator`.

Reply:
Hmm...I guess `start-mock-authenticator` is better. I will change it.

Reply:
yeah, the path has `mocks` until people look at the code. name shall be more explicit like others `build:*`, `test:*`. having said so, `start:authenticator` seems better choice.